recipe/meta.yaml

```
@@ -22,13 +22,14 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
+  number: 1
   skip: true  # [not (linux64 or aarch64)]
   skip: true  # [cuda_compiler_version in (None, "None", "") or (cuda_compiler_version or "").startswith("11")]
```

If disabling binary path replacement doesn't work, a variant on the above change is to [disable path replacement in general]( https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#binary-relocation )

```suggestion
  skip: true  # [cuda_compiler_version in (None, "None", "") or (cuda_compiler_version or "").startswith("11")]
  # Skip binary path replacement
  binary_relocation: false
```

This would also disable RPATH replacement.

So we would need to [manually update RPATHs with `patchelf`]( https://github.com/conda-forge/cuda-cudart-feedstock/blob/3194622d0c7a33782ab19ec674680de30f54c3f9/recipe/build.sh#L35 ) unless they RPATHs are already set to be relocatable